src/Native/Runtime/windows/PalRedhawkMinWin.cpp

```diff
@@ -1776,7 +1774,7 @@ struct GCThreadStubParam
 };
 
 // GC thread stub to convert GC thread function to an OS specific thread function
-static DWORD GCThreadStub(void* param)
+static DWORD REDHAWK_PALAPI GCThreadStub(void* param)
```

Comment:
This should be either explicit `__stdcall`, or removed and depend on the default.

Reply:
There is mix usage; some places have explicit __stdcall, some have `REDHAWK_PALAPI` macro, I though latter one is better.

Reply:
It depends on context. `REDHAWK_PALAPI` is better in some places, `__stdcall` is better in other places - like here where the calling convention is required by Windows SDK.